SmartDeviceLink/public/SDLSystemCapabilityManager.m

```diff
@@ -201,7 +201,13 @@ - (nullable SDLWindowCapability *)defaultMainWindowCapability {
     }
 
     // Copy all available display capability properties
-    defaultWindowCapability.templatesAvailable = [self.displayCapabilities.templatesAvailable copy];
+    NSArray *templatesArray = [self.displayCapabilities.templatesAvailable copy];
+    if ([templatesArray containsObject:@"NON_MEDIA"]) {
+        NSMutableArray *templatesMutableArray = [templatesArray mutableCopy];
+        [templatesMutableArray addObject:@"NON-MEDIA"];
+        templatesArray = [NSArray arrayWithArray:templatesMutableArray];
+    }
```

Comment:
Use `indexOfObject:` instead to get an index that `NON_MEDIA` is at (if it's in there), then use `replaceObjectAtIndex:withObject:` to replace `NON-MEDIA` instead of merely adding it.